doc/chapters/gateways/index.rst

```diff
@@ -302,12 +302,39 @@ An example configuration can look like this::
 Network gateway
 ===============
 
-The Network Gateway is used to provide Internet access through a container-specific firewall.
+The Network Gateway is used to setup network connection and configure which traffic is allowed and not.
 
 Configuration
 -------------
+Types: "INCOMING" and "OUTGOING"
+Targets: "ACCEPT", "DROP" and "REJECT"
+host: hostname or ip-address with netmask
+default: target to choose when not matching any other rule
 
-T.B.D.
+Example of network gateway config::
```

Comment:
As we discussed offline, there will be a need for some kind of priority attribute for each entry. Otherwise app developers need to specify their capabilities in a specific order.